selfdrive/car/ford/tests/test_ford.py

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env python3
+import unittest
+from parameterized import parameterized
+from typing import Dict, Iterable, Optional, Tuple
+
+import capnp
+
+from cereal import car
+from selfdrive.car.ford.values import CAR, FW_QUERY_CONFIG, FW_VERSIONS
+
+Ecu = car.CarParams.Ecu
+
+
+ECU_ADDRESSES = {
+  Ecu.eps: 0x730,          # Power Steering Control Module (PSCM)
+  Ecu.abs: 0x760,          # Anti-Lock Brake System (ABS)
+  Ecu.fwdRadar: 0x764,     # Cruise Control Module (CCM)
+  Ecu.fwdCamera: 0x706,    # Image Processing Module A (IPMA)
+  Ecu.engine: 0x7E0,       # Powertrain Control Module (PCM)
+  Ecu.shiftByWire: 0x732,  # Gear Shift Module (GSM)
+}
+
+
+ECU_FW_CORE = {
+  Ecu.eps: [
+    "14D003",
+  ],
+  Ecu.abs: [
+    "2D053",
+  ],
+  Ecu.fwdRadar: [
+    "14D049",
+  ],
+  Ecu.fwdCamera: [
+    "14F397",  # Ford Q3
+    "14H102",  # Ford Q4
+  ],
+  Ecu.engine: [
+    "14C204",
+  ],
+}
+
+
+class TestFordFW(unittest.TestCase):
+  def test_fw_query_config(self):
+    for (ecu, addr, subaddr) in FW_QUERY_CONFIG.extra_ecus:
+      self.assertIn(ecu, ECU_ADDRESSES, "Unknown ECU")
+      self.assertEqual(addr, ECU_ADDRESSES[ecu], "ECU address mismatch")
+      self.assertIsNone(subaddr, "Unexpected ECU subaddress")
```

Comment:
For this kind of test, I think it should be explicitly in the FwQueryConfig at some point, defining all your Ecus that you are able to use, we don't test this anywhere else right now, so don't know if we need it